README.rst

```diff
@@ -66,7 +72,7 @@ Get started with tutorials
       <li> <b>Dec 2020 🤖</b>  <code>cleanlab</code>` supports <a href="https://github.com/cleanlab/cleanlab#pu-learning-with-cleanlab">PU learning</a>.</li>
       <li> <b>Feb 2020 🤖</b>  <code>cleanlab</code> now natively supports Mac, Linux, and Windows.</li>
       <li> <b>Feb 2020 🤖</b>  <code>cleanlab</code> now supports <a href="https://github.com/cleanlab/cleanlab/blob/master/cleanlab/coteaching.py">Co-Teaching</a> <a href="https://arxiv.org/abs/1804.06872">(Han et al., 2018)</a>.</li>
-      <li> <b>Jan 2020 🎉</b> <code>cleanlab</code> achieves state-of-the-art on CIFAR-10 for learning with noisy labels. Code to reproduce is here:  <a href="https://github.com/cleanlab/cleanlab/tree/master/examples/cifar10">examples/cifar10</a>. This is a great place for newcomers to see how to use cleanlab on real datasets. Data needed is available in the <a href="https://github.com/cgnorthcutt/confidentlearning-reproduce">confidentlearning-reproduce</a> repo, <code>cleanlab</code> v0.1.0 reproduces results in <a href="https://arxiv.org/abs/1911.00068">the CL paper</a>.</li>
+      <li> <b>Jan 2020 🎉</b> <code>cleanlab</code> achieves state-of-the-art on CIFAR-10 with noisy labels. Code to reproduce:  <a href="https://github.com/cleanlab/examples/tree/master/cifar10">examples/cifar10</a>. This is a great place to see how to use cleanlab on real datasets.
```

Comment:
Clarified precomputed predicted-probas are provided there for you.  I don't think confidentlearning-reproduce should be linked here since it is the first thing user sees when they go to: https://github.com/cleanlab/examples/tree/master/cifar10